cms/djangoapps/contentstore/views/item.py

```diff
@@ -89,6 +89,9 @@
 NEVER = lambda x: False
 ALWAYS = lambda x: True
 
+@pluggable_override("PROPAGATE_LIBRARY_UPDATES")
```

Comment:
Pluggable override names should always be prefixed with `OVERRIDE_`

```suggestion
@pluggable_override("OVERRIDE_PROPAGATE_LIBRARY_UPDATES")
```